src/edu/washington/escience/myria/JsonTupleWriter.java

```diff
@@ -50,6 +53,11 @@ public JsonTupleWriter(final OutputStream output) {
     this.output = new PrintWriter(new BufferedWriter(new OutputStreamWriter(output)));
```

Comment:
As above, shouldn't we remove this constructor if we expect callers to supply the `OutputStream` in `open()`?